src/index.ts

```diff
@@ -1,10 +1,7 @@
 import { initialize, loggers } from '@projecttacoma/node-fhir-server-core';
-import * as dotenv from 'dotenv';
 import { serverConfig } from './config/serverConfig';
 import { Connection } from './db/Connection';
 
-dotenv.config();
```

Comment:
why did this get yoinked from index.ts and into the db connection? I feel like we want to call dotenv as far up as possible (i.e. the entrypoint of the server).

Reply:
I thought that if we import Connection here, and the envConfig file was imported in Connection it would actually get run before this line is executed... is that not the case? Or, by far up, do you mean as close to the code that starts the server as possible. This was also convenient for testing purposes since we could just import Connection in our test utils file and get envConfig to run along with it, but that can be changed by just importing the envConfig there if you think it makes sense to keep `dotenv.config()` in this file.

Reply:
Bonus: we can also silence the logger in this file as shown in the example.

I think I stole this code from you, Nat. Lol